### PR TITLE
add support for the original squad format

### PR DESCRIPTION
**Pull request by bhavani105**
# Pull Request      

## What does this PR do?

Add support for the original SQUAD json format
Added test case 

Notes:
- Replace `(issue)` **above** ↑↑↑ with the issue this PR closes to automatically link the two.
This must be done when the PR is created.
- Add multiple `Closes #(issue)` as needed.
- If this PR is work towards but does not close an issue, simply tag the issue without mentioning `Closes`.

## Description

**Describe** the changes proposed by this PR below to give the reviewer context below ↓↓↓

```(description)```     

### Request Review

Be sure to **request a review** from one or more reviewers (unless the PR is to an unprotected branch).

## Versioning

When opening a PR to make changes to PrimeQA (i.e. `primeqa/`) master, be sure to increment the version following
[semantic versioning](https://semver.org/).  The VERSION is stored [here](https://github.com/primeqa/primeqa/blob/main/VERSION) 
and is incremented using `bump2version {patch,minor,major}` as described in the (development guide documentation)[development.html].

 - [ ] Have you updated the VERSION?
 - [ ] Or does this PR not change the `primeqa` package or was not into master?

After pulling in changes from master to an existing PR, ensure the VERSION is updated appropriately.
This may require bumping the version again if it has been previously bumped.

If you're not quite ready yet to post a PR for review, feel free to open a [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/).

### After Merging
If merging into master and VERSION was updated, after this PR is merged:
- [ ] [Create a release](https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release) from the master with version equal to [VERSION](https://github.com/primeqa/primeqa/blob/main/VERSION)
- [ ] Not merging to master or VERSION not updated

### Checklist

Review the following and mark as completed:
- [ ] [Tag an issue or issues](#what-does-this-pr-do) this PR addresses.
- [ ] Added [description](#description) of changes proposed.
- [ ] Review requested as [appropriate](#request-review).
- [ ] Version bumped as [appropriate](#Versioning).
- [ ] New classes, methods, and functions documented.
- [ ] Documentation for modified code is updated.
- [ ] Built documentation to confirm it renders as expected (see [here](development.md)).
- [ ] Code cleaned up and commented out code removed.
- [ ] Tests added to ensure all functionalities tested at >= 60% unit test coverage (see [here](development.md)).
- [ ] Code cleaned up and commented out code removed.
- [ ] Release created [as needed](#after-merging) after merging.

### CODE REVIEW

**Review by seirasto — COMMENTED** [review]
Changes look good. Can you please add some comments? Should we also add that we have this functionality in the README?

**Comment by bhavani105** [comment]
Thanks @seirasto I have added comments in the README and the preprocessor

**Review by seirasto — APPROVED** [review]
LGTM

File: primeqa/mrc/README.md [hunk 1]
```diff
@@ -153,14 +153,29 @@ python primeqa/mrc/run_mrc.py --model_name_or_path ${TRAINING_OUTPUT_DIR} \
...
 ```
+For the SQUAD 1.1 dataset in the original Stanford format, first download the train and dev files from here:
+
+- https://rajpurkar.github.io/SQuAD-explorer/dataset/train-v1.1.json
+- https://rajpurkar.github.io/SQuAD-explorer/dataset/dev-v1.1.json
+
+Use the following additional command line arguments for train and eval:
+
+```
+       --train_file <path-to-train-v1.1.json> \
```
[the remark this answers is not in the document]
Comment (bhavani105) [reply primeqa/mrc/README.md:172]: yes, tested train and eval using the downloaded files in Stanford format

File: primeqa/mrc/README.md [hunk 2]
```diff
@@ -153,14 +153,29 @@ python primeqa/mrc/run_mrc.py --model_name_or_path ${TRAINING_OUTPUT_DIR} \
...
+       --eval_file <path-to-dev-v1.1.json> \
```
[the remark this answers is not in the document]
Comment (bhavani105) [reply primeqa/mrc/README.md:173]: yes tested

**Review by rongzhangibm — APPROVED** [review]
LGTM

**Review by ma-sultan — APPROVED** [review]
LGTM

**Pull request merged**